market_maker_keeper/dydx_market_maker_keeper.py

```diff
@@ -112,7 +112,7 @@ def our_available_balance(self, our_balances: dict, token: str) -> Wad:
 
     def _should_place_order(self, new_order: dict) -> bool:
         amount = new_order.pay_amount if new_order.is_sell else new_order.buy_amount
-        decimal_exponent = int(self.market_info[self.pair().upper()]['baseCurrency']['decimals'])
+        decimal_exponent = int(self.market_info[self.pair().upper()]['quoteCurrency']['decimals'])
```

Comment:
Shouldn't it be `baseCurrency` for buys and `quoteCurrency` for sells?

Reply:
yeah, ended up just removing that and simplifying that logic since minimumOrderSize as returned from DyDx always has the same number of decimals.